### PR TITLE
Press Enter key to insert item

### PR DESCRIPTION
**Pull request by variable5707**
this may resolve the issue press enter key to trigger an operation

### CODE REVIEW

**Comment by ranndev** [comment]
Welcome @variable5707 and thank you so much for contributing! :tada:

**Comment by ranndev** [comment]
Your code is still not satisfy the issue yet. Can you also support **Enter** key on **Change**, **Delete** and **Toggle** operations from the form?

File: todos.js [hunk 1]
```diff
@@ -73,11 +73,24 @@ var todoList = {
 	}
 };
 
+var input = document.getElementById("todoAddTextInput");
+input.addEventListener("keyup", function(event) {
+  if (event.keyCode === 13) {
+    event.preventDefault();
+    document.getElementById("btn").click();
```
Comment (blacksag) [inline todos.js:80]: To have a cleaner code this line (80) should be replaced with calling of `handler.addTodos()`. Also this whole section (line 76-82) should be at end with appropriate comments.

File: todos.js [hunk 2]
```diff
@@ -73,11 +73,24 @@ var todoList = {
...
+
+
+
 var handler = {
 	size : 0,
 	
-	addTodos : function() {
+	
+	
+	addTodos : function(e) {
```
Comment (blacksag) [inline todos.js:91]: The argument `e`  creates bug wherever this function is called, the argument is irrelevant and should be removed.

File: todos.js [hunk 3]
```diff
@@ -73,11 +73,24 @@ var todoList = {
...
+  }
+});
+
+
+
 var handler = {
 	size : 0,
 	
-	addTodos : function() {
+
```
Comment (blacksag) [inline todos.js:89]: remove white-space! (extra new blank line 89, 90, 93, 105, 106 & 107)

**Review by blacksag — CHANGES_REQUESTED** [review]
Do the required changes and you are good to go

**Comment by blacksag** [comment]
> Your code is still not satisfy the issue yet. Can you also support **Enter** key on **Change**, **Delete** and **Toggle** operations from the form?

I think this PR can only have **Enter** feature for adding items. Further work can be done to complete the issue.

**Comment by rannie-peralta** [comment]
> > Your code is still not satisfy the issue yet. Can you also support **Enter** key on **Change**, **Delete** and **Toggle** operations from the form?
> 
> I think this PR can only have **Enter** feature for adding items. Further work can be done to complete the issue.

Alright

**Comment by ranndev** [comment]
So sorry I accidentally close the PR.

Alright, will just wait for your comments to be resolved.

**Pull request closed without merging**